### ISSUE TITLE
Unable to write to temporary file

### ISSUE THREAD

**Issue by whygee-dev** [BemiHQ/BemiDB#6]
```
make sync
```

panics with the following exception:

```
devbox run --env-file .env "cd src && go run . sync"
[INFO] Syncing public.netflix_shows...

panic: ERROR: could not open file "/var/folders/60/4zht7bc16mv_5x6lbf6l8mj00000gn/T/public.netflix_shows1342941004" for writing: No such file or directory (SQLSTATE 58P01)

goroutine 1 [running]:
main.PanicIfError({0x1050569a0?, 0x1400037f500?}, {0x0?, 0x140001ca200?, 0x7e?})
        /redacted/BemiDB/src/utils.go:14 +0x9c
main.(*Syncer).exportPgTableToCsv(0x140001b85b8, 0x14000481680, {{0x1400046c018?, 0x140001ddc98?}, {0x1400046c070?, 0x10484aece?}})
        /redacted/BemiDB/src/syncer.go:193 +0x214
main.(*Syncer).syncFromPgTable(0x140001b85b8, 0x14000481680, {{0x1400046c018?, 0x6?}, {0x1400046c070?, 0x0?}})
        /redacted/BemiDB/src/syncer.go:98 +0xe0
main.(*Syncer).SyncFromPostgres(0x140001b85b8)
        /redacted/BemiDB/src/syncer.go:47 +0x1a8
main.syncFromPg(0x1056eb8a0)
        /redacted/BemiDB/src/main.go:60 +0x24
main.main()
        /redacted/BemiDB/src/main.go:25 +0xb0
exit status 2
```

I have tried checking the existance of the temporary file with

```go
if _, err := os.Stat(tempFile.Name()); errors.Is(err, os.ErrNotExist) {
  println("Temporary file does not exist")
}  else {
  println("Temporary file exists")
}
```

it does exist, this may indicate that pg is unable to write to this location

**Comment by whygee-dev** [comment]
Found the problem.

The following 
```go
result, err := conn.Exec(
  context.Background(),
  "COPY "+pgSchemaTable.String()+" TO '"+tempFile.Name()+"' WITH CSV HEADER",
)
```
assumes that PostgreSQL is running in the host machine which does not work if it is running in a docker container

**Comment by exAspArk** [comment]
Thanks for sharing this, @whygee-dev! 

I think this PR https://github.com/BemiHQ/BemiDB/pull/8 (released in v0.3.0) has resolved this issue.
